test/utils/fit-bounds.spec.js

```diff
@@ -1,20 +1,72 @@
 import test from 'tape-catch';
 import {fitBounds} from 'react-map-gl';
 
+/* fitBounds from Mapbox for comparison */
+import Transform from 'mapbox-gl/js/geo/transform';
```

Comment:
Does this introduce dependency on versions of mapbox that still export this? We're pretty behind with their public release.